platform/mellanox/issu-version.mk

```diff
@@ -18,7 +18,13 @@
 
 ISSU_VERSION_FILE = issu-version
 $(ISSU_VERSION_FILE)_SRC_PATH = $(PLATFORM_PATH)/issu-version
+
+ifeq ($(BLDENV), buster)
```

Comment:
>buster [](http://example.com/codeflow?start=17&length=6)

Wondering what is the DEPENDS for debian version before buster and after buster?

Reply:
Can you please elaborate on your question?

This line exists because this only depends on APPLIBS when the build environment is buster (when it is actually generating the issu_version file) 

Otherwise it will try to compile the mellanox sdk for bullseye (which it is not ready to do right now)

Reply:
Just to double check: you mean issu_version only exists in Buster, Not in Bulleyes and later, not in Stretch and earlier versions?